File: JAVA/idade.java
import java.util.Scanner;

class idade {

    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Qual sua idade?");
        int num1 = scanner.nextInt();

        String resultado = (num1 < 18) ? "Você é menor de idade." : "Você é maior de idade";
        System.out.println(resultado);
    }
}

File: C/idade.c
#include <stdio.h>

// Solicita a operação
int main(){
    int idade;
    printf("Qual a sua idade?\n");
    scanf("%d", &idade);

    printf("Voce e %s\n", (idade < 18  ? "Menor de idade":"Maior de idade"));

    float num1;
    printf("Digite qualquer coisa para encerrar o programa. \n");
    scanf("%s", &num1);
    return 0;
}